SalesforceCommerceCloud/pwa-kit
# [Release process] Changelog files are updated (@W-16164410@)

# Description



The PR adds the `check-changelog` action that checks if any Changelog file is updated for the packages that have changes filed to enforce updating Changelog.md file.

The new label "skip changelog" allows to skip the "Changelog Check" GA step even if Changelog.md files are not updated.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Add `check-changelog` action to GA workflow to enforce updating Changelog.md files.

# How to Test-Drive This PR

Check the GA builds for the tested scenarios:
1. FAIL Test Commit without updating the Changelog.md file: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1948/commits/ea6f89eeaec9057fdaf2f29ffff390d82ac9f40e

2. FAIL Test Commit without updating the Changelog.md file: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1948/commits/f7723a98ec2582cf8f9c12a1625ffdd61e0a4f38

3. PASS Test Commit without updating the Changelog.md file and the `skip changelog` label: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1948/commits/41a64232317917f24feb3e4a36eaa8f757a29088

4. PASS Test Commit with changes on non-public packages: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1948/commits/a543df01d796dc446416caa6e58642dd45f00689



- Feel free to branch off `ga-check-changelog` base branch and create your own PR for further testing.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
